config/values/finance.yml

```diff
@@ -1,19 +1,19 @@
 finance:
-  adultdependantsgrant:
+   adultdependantsgrant:
     value: £3,438
-    incomethreshold: £15,621.98
-  ## implemented above this line
+    income: £15,621.98
   childcaregrantweeklypayment:
-    onechild: £193.62
-    twoormorechildren: £331.95
+    one: £193.62
+    twoplus: £331.95
   disabilityadditionalfunding: £26,948
-  maintenanceloanpaymentvalue:
-    liveathomewithparents: £8,610
-    awayfromparentsoutsidelondon: £10,227
-    awayfromparentsinsidelondon: £13,348
+  maintenanceloanvalue:
```

Comment:
do we even need `value` on the end of this?

Reply:
Agree, will amend now